engine/moduleManager/modules/input/KeyEvent.hpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by 11896 on 26/11/2024.
+//
+
+#ifndef BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_INPUT_KEYEVENT_HPP_
+#define BLOCKYENGINE_ENGINE_MODULEMANAGER_MODULES_INPUT_KEYEVENT_HPP_
+
+#include "SDL_keycode.h"
+#include "KeyInput.hpp"
+
+enum class KeyState {
+	KEY_UP,
+	KEY_DOWN
+};
+
+struct KeyEvent {
+	KeyInput key;
+	KeyState state;
+
+	KeyEvent(KeyInput key, KeyState state)
```

Comment:
zo'n default constructor is overbodig geloof ik als je een struct gebruikt? maar t mag er voor de duidelijkheid prima in hoor